frb_codegen/src/generator/rust/ty_struct.rs

```diff
@@ -23,9 +23,17 @@ impl TypeRustGeneratorTrait for TypeStructRefGenerator<'_> {
                     String::new()
                 };
 
+                let shared_mod_name = self.get_shared_module_of_a_type(&field.ty);
                 Acc {
+                    io: if !self.context.config.shared && shared_mod_name.is_some() {
+                        format!(
+                            "{field_} {}::Wire2Api::wire2api(self.{field_name})",
```

Comment:
nit: alternative may be

`<self.{field_name} as {mod_name}::Wire2Api>.wire2api()`

choose whichever you like

Reply:
For stability, I am gonna keep no change as possible as I can.